### PR TITLE
August Patch

### PR DESCRIPTION
**Pull request by Pippow**
Here is half the cards approved in the August review process:

**New Cards:**
- Tuirseach Warship: 3pt Tuirseach Machine - Deploy: Damage an ennemy unit by this card's base strength.
- Dwarf Miner: 6pt Dwarf - Deploy: Strengthen self by 1 for every Dwarf in your hand.
- Vrihedd Saboteur: 1pt Elf, Soldier - Deploy: Boost self by 1 for each Elf remaining in your deck.

**Reworks**
- Gascon: Deploy: Choose a row and move all enemies on it to random rows and Damage self by 1 for each unit moved.\nWhen in your hand or deck, Boost self by 1 whenever a unit moves during your turn.

The rain cards are suspended until the main rain archetype is fixed/

### CODE REVIEW

**Comment by Pippow** [comment]
Hi @NPC-YFY I tested the changes you approved locally, my feedback is 

- 矮人矿工 and 维里赫德旅破坏者 are too weak and would need some more points
- The 图尔赛克战船 should be a soldier not a ship, to work with priestesses of Freia (corsairs don't fit well in veterans)
- The 矮人矿工 should be a soldier because there is no point in strenghtening them if they cannot be resurected with feign death

你好 @NPC-YFY 我在本地测试了您批准的修改，我的反馈是 

- 矮人矿工和维里赫德旅破坏者太弱了，需要更多的点数。
- 图尔赛克战船应该是士兵而不是战舰，以便与弗蕾亚女祭司合作（海盗不适合老兵）
- 矮人矿工应该是士兵，因为如果他们不能通过装死复活，那么强化他们就没有意义了。

**Comment by Pippow** [comment]
@NPC-YFY also an important question: should cards that move themselves, like Vysogota, Archespores, Whale etc. trigger Gascon? There have been many questions about it on Discord asking if it is a bug and asked for such a change.

**Comment by neal2018** [comment]
self reminder: plus 1 on gwentmap version before merge

File: src/Cynthia.Card/src/Cynthia.Card.Server/Locales/pl.json [hunk 1]
```diff
@@ -3083,7 +3083,7 @@
           },
           "70032": {
               "Name": "Gascon",
-              "Info": "Rozmieszczenie: Przenieś wszystkie wrogie jednostki do losowych rzędów. Za każdą przeniesioną jednostkę otrzymaj 2 pkt obrażeń. Wzmocnij się o 1 za każdym razem gdy jednostka zostanie poruszona w trakcie Twojej tury.",
```
Comment (neal2018) [inline src/Cynthia.Card/src/Cynthia.Card.Server/Locales/pl.json:3086]: This remove is not intended, is it?

Comment (Pippow) [reply src/Cynthia.Card/src/Cynthia.Card.Server/Locales/pl.json:3086]: The removal was intended but I forgot to put back the new description, thanks for noticing, I fixed it.

**Comment by Pippow** [comment]
If anyone is reading, I just realized I made Gascon move only units on the opposing row, I need to make him move both.
Also it was said on QQ that the card should move the row with most units. It's slightly unclear : ideally we should count the sums of each row'unit count on both sides and go for the highest value among the 3. But this is quite counterintuitive, another possible implementation would be to go for any of the 6 rows with most units, but as the card should affect both sides it's weird too.

**Comment by Pippow** [comment]
@neal2018 would you mind helping me on the Gascon change? I tried to implement the feature "move the units on both sides of the row with most units" following the creator's design, but I couldn't manage to do it. I don't really understand how rows are managed in the code. Thanks! This is the only bit that remains and after we can put this batch behind us.

**Comment by Pippow** [comment]
@neal2018 Can you please merge it? I think I managed to make Gascon work, tried a couple scenarios and it seemed to work; but it would be nice if you could double check Gascon. And don't forget to update the gwentmap count. Thanks a lot.

**Pull request merged**